### PR TITLE
Update invite email

### PR DESCRIPTION
**Pull request by d8660091**
- Add expire time
- Add platform link

### CODE REVIEW

**Comment by d8660091** [comment]
We need to add platformUrl variable to the helm values

File: src/main/java/org/icgc/argo/program_service/services/MailService.java [hunk 1]
```diff
@@ -46,14 +48,17 @@
   @Value("${app.invitationUrlPrefix}")
   private String invitationUrlPrefix;
 
+  @Value("${app.platformUrl}")
+  private String platformUrl;
```
Comment (andricDu) [inline src/main/java/org/icgc/argo/program_service/services/MailService.java]: :+1:

**Comment by d8660091** [comment]
@andricDu test is fixed

**Comment by andricDu** [comment]
Thanks @d8660091

**Pull request merged**